fping/README.md

```diff
@@ -15,4 +15,6 @@ Binary package
 
 ## Usage
 
-*TODO: Add instructions for usage*
+This package needs the file `/etc/protocols` from the `core/iana-etc` package:
+
+`ln -sf $(hab pkg path core/iana-etc)/etc/protocols /etc/protocols`
```

Comment:
I think we should be careful here, we don't want to accidentally clobber a system file. I'd recommend

```
if [[ ! -f /etc/protocols ]]; then
  hab pkg install core/iana-etc
  ln -s $(hab pkg path core/iana-etc)/etc/protocols /etc/protocols
fi
```